gen/injection.js

```diff
@@ -0,0 +1,166 @@
+        return;
+    }
+
+    if(!httpOptions){
+        for (var i = 0 ; i < arguments.length; i++){
+            if (isObject(arguments[i])){
+                httpOptions = arguments[i];
+            }
+        }
+    }
+    var url;
+    switch(arguments.length-1){
+        case 0:
+            url = parse('/edge/v1/admin/service/:ServiceID/config', [ServiceID]);
+            break;
+        case 1:
+            url = parse('/edge/v1/admin/service/:ServiceID/config/:ConfigSection', [ServiceID, ConfigSection]);
+            break;
+        case 2:
+            url = parse('/edge/v1/admin/service/:ServiceID/config/:ConfigSection/:ConfigKey', [ServiceID, ConfigSection, ConfigKey]);
+            break;
+        case 3:
+            url = parse('/edge/v1/admin/service/:ServiceID/config/:ConfigSection/:ConfigKey', [ServiceID,ConfigSection,ConfigKey]);
+            break;
+        default:
+            sys.logs.error('Invalid argument received.');
+            return;
+    }
+    sys.logs.debug('[veritone] GET from: ' + url);
+    var options = checkHttpOptions(url, httpOptions);
+    return endpoint._get(options);
+    //END_INJECTION
+}
+
+aiDataAdminOrganizationsOpenidRolesGet = () => {
+    //INIT_INJECTION
+    if (aguments.length === 0 || !id) {
+        sys.logs.error('Invalid argument received. This helper should receive the following parameters as non-empty strings: [id].');
+        return;
+    }
+
+    if(!httpOptions){
+        for (var i = 0 ; i < arguments.length; i++){
+            if (isObject(arguments[i])){
+                httpOptions = arguments[i];
+            }
+        }
+    }
+    var url;
+    switch(arguments.length-1){
+        case 0:
+            url = parse('/api/admin/organizations/:id/openid-roles', [id]);
+            break;
+        case 1:
+            url = parse('/api/admin/organizations/:id/openid-roles/:openidRoleKey', [id, openidRoleKey]);
+            break;
+        case 2:
+            url = parse('/api/admin/organizations/:id/openid-roles/:openidRoleKey', [id,openidRoleKey]);
+            break;
+        default:
+            sys.logs.error('Invalid argument received.');
+            return;
+    }
+    sys.logs.debug('[veritone] GET from: ' + url);
+    var options = checkHttpOptions(url, httpOptions);
+    return endpoint._get(options);
+    //END_INJECTION
+}
+
+voiceTtsVoiceGet = () => {
+    //INIT_INJECTION
+    if (aguments.length === 0 || !voiceId) {
+        sys.logs.error('Invalid argument received. This helper should receive the following parameters as non-empty strings: [httpOptions or voiceId].');
+        return;
+    }
+    if(!httpOptions){
+        for (var i = 0 ; i < arguments.length; i++){
+            if (isObject(arguments[i])){
+                httpOptions = arguments[i];
+            }
+        }
+    }
+    var url;
+    switch(arguments.length-1){
+        case 0:
+            url = parse('/v2/tts/voice?lang');
+            var options = checkHttpOptions(url, voiceId);
```

Comment:
I think you should pass httpOptions instead of voiceId.

Reply:
This is the embedded version of the previous code, same explanation.
https://github.com/slingr-stack/veritone-endpoint/pull/2#pullrequestreview-1379138435

Reply:
Its operation is explained in the commentary.
https://github.com/slingr-stack/veritone-endpoint/pull/2#pullrequestreview-1379138435

You could not send httpOptions because it is the last parameter in the argument list and would never enter case 0.